### PR TITLE
test pr build

### PR DESCRIPTION
**Pull request by 1dot75cm**

### CODE REVIEW

**Comment by 1dot75cm** [comment]
test xxx

**Comment by 1dot75cm** [comment]
Can one of the admins verify this patch?

**Comment by 1dot75cm** [comment]
ok to test

**Comment by 1dot75cm** [comment]
ok to test

File: README.md [hunk 1]
```diff
@@ -7,3 +7,7 @@
 - [Issue Tracker 问题追踪](https://github.com/FZUG/repo/issues)
 - [Code 源代码](https://github.com/FZUG/repo)
 - [Release monitoring 上游项目版本跟踪](https://release-monitoring.org/distro/FZUG)
+
+test
+test!
+test2
```
Comment (sztsian) [inline README.md]: What do you want by this 3 line @1dot75cm

Comment (1dot75cm) [inline README.md]: I try to use [ghprb-plugin](https://github.com/jenkinsci/ghprb-plugin) to build pull request. See https://repo.fdzh.org/jenkins/job/pr_builder/8/. This PR for testing. @sztsian

**Comment by 1dot75cm** [comment]
Refer to this link for build results (access rights to CI server needed): 
null/job/pr_builder/45/

**Comment by 1dot75cm** [comment]
Refer to this link for build results (access rights to CI server needed): 
null/job/pr_builder/46/

**Pull request closed without merging**